src/makeRobot.js

```diff
@@ -38,7 +38,87 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    version: `chip version: ${version}`,
+    wheels,
+
+    get info() {
+      return `name: ${this.name}, ${this.version}, wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
```

Comment:
Better change order, because initial values(like coords) better write before methods.
```suggestion
    wheels,
    
    coords: {
      x: 0,
      y: 0,
    },

    get info() {
      return `name: ${this.name}, ${this.version}, wheels: ${this.wheels}`;
    },

    get location() {
      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
    },
```